src/devices/Card/Icode/IcodeCardCapacity.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Card.Mifare
+{
+    /// <summary>
+    ///  Different storage capacity for ICODE cards
+    ///  https://www.nxp.com.cn/docs/en/application-note/AN11809.pdf
+    /// </summary>
+    public enum IcodeCardCapacity
+    {
+        /// <summary>
+        /// Unknown
+        /// </summary>
+        Unknown = 0,
+
+        /// <summary>
+        /// ICODE SLIX
+        /// </summary>
+        ICODESLIX = 896,
```

Comment:
we tend to use Pascal Case naming for publis elements. For those ones as well. So should be more like IcodeSlix, same pattern for the following ones